### DISCUSSION TITLE
Achievements

### DISCUSSION THREAD

**Discussion by Ndamix-Store** [community/community#53399] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Start your discussion!

**Comment by chrisK824** [comment]
Hope there are more to unveil than just those few so far 😋

**Comment by Jeff-JungJaeho** [comment]
bibidi babidi boo😐

**Comment by vowelparrot** [comment]
I'd love it if open source projects could create custom achievements to bestow on contributors

**Comment by Ndamix-Store** [comment]
As my pleasure and will be done consequently to the said request from yours.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by YasinDehfuli** [comment]
if you need to learn how to get github achievements, use this Step-by-Step course : 

https://github.com/4xmen/Get-Github-Achievements
